www/src/Pages/AddonsConfigPage.jsx

```diff
@@ -212,7 +212,8 @@ export default function AddonsConfigPage() {
 			onSubmit={onSuccess}
 			initialValues={defaultValues}
 		>
-			{({ handleSubmit, handleChange, values, errors, setFieldValue }) => (
+			{({ handleSubmit, handleChange, values, errors, setFieldValue }) => 
+			console.log('errors', errors) || (
```

Comment:
show error to console log for developers

Reply:
Makes sense, we don't have to worry about being too verbose!